bids/grabbids/bids_validator.py

```diff
@@ -0,0 +1,207 @@
+import re
+
+class BIDSValidator():
```

Comment:
Can you add docstrings for all the public methods in this class (and the initializer)? I know we haven't been great about doing that for the rest of the codebase, but we should probably start enforcing that now that other people are contributing.

Reply:
Sure, I can do it. I didn't do it because I wasn't sure that names of parameters and functions will stay the same after eview. Once we finalize essential parts, I will do it.

Reply:
@tyarkoni I have added the test_validator.py file to tests where testing of .is_anat() is implemented. If this format is okay, I will add tests for all functions of Validator.

Reply:
Sorry it took a while to get around to this. This was not exactly what I had in mind for the tests, but I actually like the way you structured it more than what I was thinking (which was to have both the `True` and `False`-evaluating cases in the same method, so yes, let's go with this!